Casks/f/fujifilm-x-raw-studio.rb

```diff
@@ -0,0 +1,23 @@
+cask "fujifilm-x-raw-studio" do
+  version "1.22.0"
+  sha256 "a6f72c8ca5bbe83c5a6d2bab0c64c611112c195e617518c075b2530ac065a638"
+
+  url "https://dl.fujifilm-x.com/support/software/x-raw-studio-mac1220-p2ep21rl/XRawStudio#{version.delete(".")}.dmg"
```

Comment:
```suggestion
  url "https://dl.fujifilm-x.com/support/software/x-raw-studio-mac#{version.no_dots}-p2ep21rl/XRawStudio#{version.no_dots}.dmg"
```
We will need to see if the "p2ep21rl" part of the `url` changes also.